proceeding/text/introduction.tex

```diff
@@ -10,3 +10,16 @@ \section{Introduction}
 This is what gamma-sky.net was created for.
 
 \end{itemize}
+
+
+        The field of high-energy (HE) and very-high-energy (VHE) astronomy is growing tremendously – while only
```

Comment:
De-dent this text all the way to the left?
(The lines are still pretty long and sometimes wrap in the Github fixed-width text view.